<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgeCalculator</class>
 <widget class="QWidget" name="AgeCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Age Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(114, 159, 207)</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhDigitsOnly</set>
  </property>
  <widget class="QLabel" name="AskInp">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(211, 215, 207);
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>Enter Your Birthday</string>
   </property>
  </widget>
  <widget class="QLabel" name="InpY">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(249, 244, 244)
}</string>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="InpM">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(249, 244, 244)
}</string>
   </property>
   <property name="text">
    <string>Month</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="InpD">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(249, 244, 244)
}</string>
   </property>
   <property name="text">
    <string>Day</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="OutY">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(249, 244, 244)
}</string>
   </property>
   <property name="text">
    <string>Month/s</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="OutD">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(249, 244, 244)
}</string>
   </property>
   <property name="text">
    <string>Year/s</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="OutM">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(249, 244, 244)
}</string>
   </property>
   <property name="text">
    <string>Day/s</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelR">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(211, 215, 207);
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>Your Age is :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CombY">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(203, 195, 195) ;

</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="maxVisibleItems">
    <number>7</number>
   </property>
   <item>
    <property name="text">
     <string>2020</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2019</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2018</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2017</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2016</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2015</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2014</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2013</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2012</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2011</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2010</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2009</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2008</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2007</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2006</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2005</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2004</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2003</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2002</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2001</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1999</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1998</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1997</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1996</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1995</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1994</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1993</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1992</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1991</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1990</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1989</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1988</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1987</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1986</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1985</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="CombM">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(203, 195, 195) ;
</string>
   </property>
   <property name="currentText">
    <string>January</string>
   </property>
   <property name="maxVisibleItems">
    <number>6</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>January</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>February</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>March</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>April</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>May</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>June</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>July</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>August</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>September</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>October</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>November</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>December</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="CombD">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>2</width>
     <height>2</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>2</width>
     <height>2</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(203, 195, 195) ;
</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="maxVisibleItems">
    <number>10</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="minimumContentsLength">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>01</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>02</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>03</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>04</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>05</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>06</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>07</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>08</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>09</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>26</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>27</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>28</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>29</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>31</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="LabelY">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(203, 195, 195) ;
	border: 1px solid gray;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelM">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(203, 195, 195) ;
	border: 1px solid gray;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelD">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(203, 195, 195) ;
	border: 1px solid gray;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_res">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
